plasmapy/physics/parameters.py

```diff
@@ -674,29 +664,34 @@ def collision_rate_ion_ion(T_i, n_i, ion_particle,
     .. [3] Callen Chapter 2, http://homepages.cae.wisc.edu/~callen/chap2.pdf
 
     """
-    from plasmapy.physics.transport import Coulomb_logarithm
+    from plasmapy.physics.transport.collisions import Coulomb_logarithm
     T_i = T_i.to(u.K, equivalencies=u.temperature_energy())
     if coulomb_log is not None:
         coulomb_log_val = coulomb_log
     else:
         particles = [ion_particle, ion_particle]
-        coulomb_log_val = Coulomb_logarithm(T_i, n_i, particles, V)
+        coulomb_log_val = Coulomb_logarithm(T_i, n_i, particles, V, method=coulomb_log_method)
     Z_i = atomic.integer_charge(ion_particle)
     m_i = atomic.ion_mass(ion_particle)
-    nu_i = 4 / 3 * np.sqrt(np.pi / m_i) / (4 * np.pi * eps0)**2 * e**4 * \
-        n_i * Z_i**4 * coulomb_log_val / (k_B * T_i)**1.5
+    nu_i = 4 / 3 * np.sqrt(np.pi / m_i) / (4 * np.pi * eps0) ** 2 * e ** 4 * \
```

Comment:
Can this be cleaned up to remove the continuation slash?

Reply:
I tried replacing a bunch of those with reasonable, physical variables (`coulomb_constant` and so on) and realized I'm not making this calculation any shorter... I think this may be one of those places where it's reasonable to do this.

Reply:
I've just checked this against the definition of `collision_frequency` in `transport.py`, there is just a constant difference between them so we can rewrite it as follows. Just make sure none of my variable names collide with names in this function.

```
# ion thermal velocity (most probable)
v_thi = np.sqrt(2 * k_B * T_i / m_i)
# this is the same as b_perp in collisions.py, using most probable thermal velocity for V
# and using ion mass instead of reduced mass
bperp = (Z_i * e) ** 2 / (4 * np.pi * eps0 * m_i * v_thi ** 2)
# collisional cross-section
sigma = np.pi * (2 * bperp) ** 2
# collisional frequency with Coulomb logarithm to correct for small angle collisions
nu = n_i * sigma * v_thi * coulomb_log_val
# this coefficient is the constant that pops out when comparing this definition of 
# collisional frequency to the one in collisions.py
coeff = np.sqrt(8 / np.pi) / 3
# collisional frequency modified by the constant difference
nu_i = coeff * nu
```

If you rewrite it like this then we can double check that my math is correct. If it is then we just get rid of everything and replace it with `nu_i = coeff * nu` and call `collision_frequency` to get `nu`. Just need to make sure to have the right arguments going into `collision_frequency` such that reduced mass and V map to `m_i` and `v_thi`.

Reply:
Try compensating for `reduced_mass` which causes mass from identical particles to become `m/2` instead of `m`. This then makes `bperp -> 2 * bperp`. So if we do
```sigma = np.pi * (0.5 * 2 * bperp) ** 2```
instead of 
```sigma = np.pi * (2 * bperp) ** 2```
This should compensate for the different mass definitions.
I would leave  a comment stating that the `0.5` is compensating for reduced mass.

Reply:
Ignore the previous comment. We're supposed to be correcting `nu` not `sigma`.
The reduced mass affects `nu` as well. So do this:
```
coeff = np.sqrt(8 / np.pi) / 3 / 4
```